bin/docker-image-tool.sh

```diff
@@ -97,12 +103,13 @@ Commands:
   push        Push a pre-built image to a registry. Requires a repository address to be provided.
 
 Options:
-  -f file     Dockerfile to build for JVM based Jobs. By default builds the Dockerfile shipped with Spark.
-  -p file     Dockerfile with Python baked in. By default builds the Dockerfile shipped with Spark.
-  -r repo     Repository address.
-  -t tag      Tag to apply to the built image, or to identify the image to be pushed.
-  -m          Use minikube's Docker daemon.
-  -n          Build docker image with --no-cache
+  -f file               Dockerfile to build for JVM based Jobs. By default builds the Dockerfile shipped with Spark.
+  -p file               Dockerfile with Python baked in. By default builds the Dockerfile shipped with Spark.
```

Comment:
nit: Maybe for -p & -R clarify that it build's the Python or R docker file shipped with Spark since we are going to be shipping multiple ones and if someone goes and looks at the default one they might get confused about how this builds.